examples/run-strands-safely-on-separate-threads/run_strands_safely_on_separate_threads.md

```diff
@@ -0,0 +1,11 @@
+# Run strands safely on a separate thread based on isolation
+
+Ballerina extended the isolated feature to identify cases in which strands created by a start action, or a named worker can be run safely on separate threads.
```

Comment:
We don't need to say that we extended the functionality since this is a BBE?
```suggestion
The isolated feature in Ballerina supports identifying cases in which strands created by a start action, or a named worker can be run safely on separate threads.
```